src/main/java/fi/metatavu/famifarm/packings/CutPackingController.java

```diff
@@ -0,0 +1,161 @@
+package fi.metatavu.famifarm.packings;
+
+import fi.metatavu.famifarm.persistence.dao.CutPackingDAO;
+import fi.metatavu.famifarm.persistence.dao.ProductDAO;
+import fi.metatavu.famifarm.persistence.dao.ProductionLineDAO;
+import fi.metatavu.famifarm.persistence.model.CutPacking;
+import fi.metatavu.famifarm.persistence.model.Product;
+import fi.metatavu.famifarm.persistence.model.ProductionLine;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.UUID;
+
+@ApplicationScoped
+public class CutPackingController {
+    @Inject
+    private CutPackingDAO cutPackingDAO;
+
+    @Inject
+    private ProductDAO productDAO;
+
+    @Inject
+    private ProductionLineDAO productionLineDAO;
+
+    /**
+     * Creates
+     *
+     * @param productId the id of a product to which this cut packing belongs
+     * @param productionLineId the id of a production line to which this cut packing belongs
+     * @param weight weight in kilograms
+     * @param sowingDay a day on which these plants were sowed
+     * @param cuttingDay a day on which these plants were cut
+     * @param producer producer name
+     * @param contactInformation contact information
+     * @param gutterCount gutter count
+     * @param gutterHoleCount gutter hole count
+     * @param storageCondition storage condition
+     * @param creatorId the id of the user who is creating this cut packing
+     */
+    public CutPacking create (
+            UUID productId,
+            UUID productionLineId,
+            double weight,
+            OffsetDateTime sowingDay,
+            OffsetDateTime cuttingDay,
+            String producer,
+            String contactInformation,
+            int gutterCount,
+            int gutterHoleCount,
+            String storageCondition,
+            UUID creatorId) throws CutPackingInvalidParametersException {
+
+        Product product = productDAO.findById(productId);
+        ProductionLine productionLine = productionLineDAO.findById(productionLineId);
+
+        if (product == null) {
+            throw new CutPackingInvalidParametersException("Product with id " + productId + " not found!");
+        }
+
+        if (productionLine == null) {
+            throw new CutPackingInvalidParametersException("Production line with id " + productionLineId + " not found!");
```

Comment:
Use formatter instead of concat i.e. String.format("Product with id %s not found!", productId)